CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Unreleased
 
+  - Modify setup.py to explicitly support Py3.5 and above (PR [#471](https://github.com/RaRe-Technologies/smart_open/pull/471), [@Amertz08](https://github.com/Amertz08))
```

Comment:
Modify setup.py to explicitly support **only** Py3.5 and above

Reply:
OK, done here: https://github.com/RaRe-Technologies/smart_open/blob/develop/CHANGELOG.md